packages/ui/docs/GettingStarted.stories.mdx

```diff
@@ -0,0 +1,103 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="GettingStart" />
+
+# Getting Start
+
+With NorthStar, you can easily design and develop your UI applications.
+
+## Before you start
```

Comment:
This all seems duplicated from the readme.

Reply:
Yes. this storybook page will be displayed on the documentation website. README under the ui package will be built into the NPM package home. Not sure how to reuse either of them while avoiding redirecting users around.